src/main/java/org/jvnet/jenkins/plugins/nodelabelparameter/parameterizedtrigger/NodeLabelBuildParameter.java

```diff
@@ -38,12 +38,12 @@ public Action getAction(AbstractBuild<?, ?> build, TaskListener listener) throws
             labelExpanded = TokenMacro.expandAll(build, listener, labelExpanded);
         } catch (MacroEvaluationException e) {
             labelExpanded = nodeLabel;
+            listener.getLogger().println("Token expansion failed.");
```

Comment:
The code is already printing a full stack trace in case of macro expansion failure.  How does the extra text help the user?
```suggestion
```